[ews-app] Generate Github comment text for a given commit id

aj062 opened the pull request:
#### 440eea9e48a1d9176f6a11dfd65c750163192409
<pre>
[ews-app] Generate Github comment text for a given commit id
<a href="https://bugs.webkit.org/show_bug.cgi?id=243168">https://bugs.webkit.org/show_bug.cgi?id=243168</a>

Reviewed by Ryan Haddad.

* Tools/CISupport/ews-app/ews/common/github.py:
* Tools/CISupport/ews-app/ews/common/buildbot.py:
* Tools/CISupport/ews-app/ews/views/statusbubble.py:

Canonical link: <a href="https://commits.webkit.org/252790@main">https://commits.webkit.org/252790@main</a>
</pre>

ryanhaddad commented on Tools/CISupport/ews-app/ews/common/github.py:
```diff
@@ -131,7 +136,84 @@ def update_or_leave_comment_on_pr(self, pr_number, content, repository_url=None,
+        url = 'https://{}/#/builders/{}/builds/{}'.format(config.BUILDBOT_SERVER_HOST, build.builder_id, build.number)
+
+        if build.result is None:
+            hover_over_text = 'Build is in-progress'
```
I think this should be "in progress" without the hyphen

aproskuryakov commented on Tools/CISupport/ews-app/ews/common/buildbot.py:
```diff
@@ -169,3 +169,13 @@ def retry_build(cls, builder_id, build_number):
+
+    @classmethod
+    def is_builder_queue(self, queue):
+        icon = Buildbot.icons_for_queues_mapping.get(queue)
```
Shouldn't be a blocker for landing, but this looks like a roundabout approach. This makes a model level decision (which queue it is) from presentation data (what icon it has).

aj062 committed:
[ews-app] Generate Github comment text for a given commit id
https://bugs.webkit.org/show_bug.cgi?id=243168

Reviewed by Ryan Haddad.

* Tools/CISupport/ews-app/ews/common/github.py:
* Tools/CISupport/ews-app/ews/common/buildbot.py:
* Tools/CISupport/ews-app/ews/views/statusbubble.py:

Canonical link: https://commits.webkit.org/252790@main
